seed/seed20.go

```diff
@@ -496,34 +628,26 @@ func (s *seed20) loadEssentialMeta(filterEssential func(*asserts.ModelSnap) bool
 	essSnaps := model.EssentialSnaps()
```

Comment:
I know you didn't change it in this branch, but since you are touching this code anyway I'd warmly suggest renaming the internal `loadEssentialMeta()` function because it's not actually loading anything but only prefilling the channel with the essential snaps. Maybe `enqueueLoadMetaForEssentialSnaps()`?

Reply:
I just /load/queue/ on this and the other one that calls consider*